app/ios/PassportReader.swift

```diff
@@ -416,3 +421,38 @@ func serializePublicKey(_ publicKey: SecKey) -> String? {
     return true
   }
 }
+#else
+// E2E Testing stub implementation
```

Comment:
fyi @seshanthS added this logic to create a light iOS e2e test.

tested scanning on iOS and it still works